integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -2774,6 +2774,14 @@ msgstr "E-Mail-Adresse"
 msgid "phone number"
 msgstr "Telefonnummer"
 
+#: cms/models/contact/contact.py
+msgid "mobile phone number"
+msgstr "Mobile Telefonnummer"
```

Comment:
```suggestion
msgstr "Mobiltelefonnummer"
```

Time for the superpower of German language: just stick everything together :innocent: